build/scripts/build_helper

```diff
@@ -129,6 +129,7 @@ check_supported_distribution() {
     case "$distribution" in
         "ubuntu18.04") return 0 ;;
         "ubuntu20.04") return 0 ;;
+        "ubuntu22.04") return 0 ;;
```

Comment:
@NeptuneMagicSauce 

Add also "rhel8.7" at the end of the list. That's the reason it failed.

Reply:
After `yum update`:

```bash
$ cat /etc/os-release
NAME="Red Hat Enterprise Linux"
VERSION="8.7 (Ootpa)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="8.7"
...
```